dashboard/src/shared/specs/LoadingWrapperSharedSpecs.tsx

```diff
@@ -0,0 +1,17 @@
+import { shallow } from "enzyme";
+import * as React from "react";
+
+// Shared jest examples that checks that the provided component is rendering the Loading Wrapper
+export default (Component: any, props: any) => {
+  describe("loading spinner", () => {
+    it("renders", () => {
+      const wrapper = shallow(<Component {...props} />);
+      expect(wrapper.find("LoadingWrapper")).toExist();
```

Comment:
we should import the LoadingWrapper class and use the constant instead of a string